python/cudf/cudf/core/column/column.py

```diff
@@ -1435,6 +1437,11 @@ def _copy_type_metadata(self: T, other: ColumnBase) -> ColumnBase:
         ):
             other = other._rename_fields(self.dtype.fields.keys())
 
+        if isinstance(self.dtype, cudf.Decimal64Dtype) and isinstance(
```

Comment:
RE: the previous comment - it looks like you already know about `_copy_type_metadata` - so this should end up not being needed in the end, right?

Reply:
I believe this can just be removed now since we do `_copy_type_metadata` below.